doc/docs/2.2.x/how-tos/basic-data-operations/export-data-out-pachyderm/sql-egress.md

```diff
@@ -48,6 +48,20 @@ Append an egress section to your pipeline specification file, then fill in:
 - the `url`: the connection string to your database. Its format is identical to the [url in the SQL Ingest](../../sql-ingest/#database-connection-url){target=_blank}.
 - the `file_format` type: CSV for now.
 - the `name`: the Kubernetes secret name.
+- the `columns`: Optional array for egress of **CSV files with headers only**. The order of the columns in this array must match the order of the schema columns; however, the CSV columns can be any order. So if the array is ["foo", "bar"] and the CSV file is:
```

Comment:
I thought that JSON ingest uses the column list as well. @albscui am I wrong?

Reply:
As of today, SQL ingest uses Jsonnets. You have no obvious way to pass this column list.
The Ingest with Json uses the elements in your SELECT clause or the Aliases you are defining (i.e.: client.name as Name)